### PR TITLE
Adding role for deploying python on CoreOS

### PR DESCRIPTION
**Pull request by PiotrProkop**

### CODE REVIEW

File: roles/ceph-common-coreos/tasks/install_pypy.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- name: create local temp directory
+  local_action: raw mkdir -p {{local_temp_directory}}
+  sudo: no
+
+- name: prepare python executable
+  local_action: template src=install_python.sh.j2 dest={{local_temp_directory}}/install_python.sh
+  sudo: no
+
+- name: fix library
+  raw:  ln -s /lib64/libncurses.so.5.9 {{pypy_directory}}/lib_pypy/libtinfo.so.5
```
Comment (leseb) [inline roles/ceph-common-coreos/tasks/install_pypy.yml]: Extra space

File: roles/ceph-common-coreos/tasks/install_pypy.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- name: run install_python.sh
+  script: "{{local_temp_directory}}/install_python.sh"
+
+- name: add execute permission
+  raw:  chmod a+x {{pypy_directory}}/python
```
Comment (leseb) [inline roles/ceph-common-coreos/tasks/install_pypy.yml]: Dito

File: roles/ceph-common-coreos/tasks/install_pypy.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- name: move python to binary directory
+  raw:  mv {{pypy_directory}}/python {{pypy_binary_directory}}/python
```
Comment (leseb) [inline roles/ceph-common-coreos/tasks/install_pypy.yml]: Dito

**Comment by PiotrProkop** [comment]
Extra spaces removed

File: roles/ceph-common-coreos/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+**THIS REPOSITORY DOES NOT ACCEPT PULL REQUESTS**
+**PULL REQUESTS MUST GO THROUGH [CEPH-ANSIBLE](https://github.com/ceph/ceph-ansible)**
+
+# License
+
+Apache
+
+# Author Information
+
+This role was created by Alfredo Deza.
```
Comment (leseb) [inline roles/ceph-common-coreos/README.md]: @PiotrProkop use your name here :)

**Comment by leseb** [comment]
@PiotrProkop can you run `./generate_group_vars_sample.sh` to generate a group_vars file?

**Comment by PiotrProkop** [comment]
@leseb added common-coreos.sample

**Comment by leseb** [comment]
We should probably try to make the role a dependency of all the ceph daemons when we are deploying on CoreOS.
We can do this later. Thanks @PiotrProkop

**Pull request merged**